src/config/firebase.js

```diff
@@ -1,13 +1,24 @@
 import firebase from 'firebase/app';
 import 'firebase/auth';
+import 'firebase/analytics';
+
+import rollbar from '../shared/rollbar';
 
 const config = {
+  projectId: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
   apiKey: process.env.NEXT_PUBLIC_FIREBASE_KEY,
   authDomain: process.env.NEXT_PUBLIC_FIREBASE_DOMAIN,
+  appId: process.env.NEXT_PUBLIC_FIREBASE_APP_ID,
 };
 
-if (!firebase.apps.length) {
+if (typeof window !== 'undefined' && !firebase.apps.length) {
```

Comment:
Can use the isBrowser check in places of this typeof window.

Reply:
True! Will do that.